### PR TITLE
Create Server Entry

### PR DESCRIPTION
**Pull request by Radiumio**
**When merged this pull request will:**

- *Add appropriate code to `initPlayerLocal.sqf` to initialize the `rules.sqf`*
- *Add the script `rules.sqf` to the scripts folder for the diary entry itself.*

### CODE REVIEW

File: src/ZGM-17_Achilles_Blufor.Altis/initPlayerLocal.sqf [hunk 1]
```diff
@@ -33,4 +43,7 @@ if (_is_curator) then
 	};
 	execVM "scripts\R3F_LOG\init.sqf";
 };
-
+// initalise diary record for rulesinit
+if (hasInterface) then {
```
Comment (CreepPork) [inline src/ZGM-17_Achilles_Blufor.Altis/initPlayerLocal.sqf]: `hasInterface` shouldn't be required as it's `initPlayerLocal.sqf` so they will all be player machines with an interface.

File: src/ZGM-17_Achilles_Blufor.Altis/initPlayerLocal.sqf [hunk 2]
```diff
@@ -33,4 +43,7 @@ if (_is_curator) then
...
+	0 = [] execVM "scripts\rules.sqf";
```
Comment (CreepPork) [inline src/ZGM-17_Achilles_Blufor.Altis/initPlayerLocal.sqf]:
```suggestion
	execVM "scripts\rules.sqf";
```

File: src/ZGM-17_Achilles_Blufor.Altis/scripts/rules.sqf [hunk 3]
```diff
@@ -0,0 +1,24 @@
+waitUntil {!isNull player};
```
Comment (CreepPork) [inline src/ZGM-17_Achilles_Blufor.Altis/scripts/rules.sqf:1]:
```suggestion
waitUntil {!isNull player};

```

File: src/ZGM-17_Achilles_Blufor.Altis/initPlayerLocal.sqf [hunk 4]
```diff
@@ -11,7 +11,17 @@ if (_is_curator) then
 {
 	// set random position for curator camera
 	waitUntil {not isNull curatorCamera and {not isNil {bis_curator getVariable "CAM_POS_ATL"}}};
-	private _cam_pos = bis_curator getVariable "CAM_POS_ATL";
+	private _respawns = allMissionObjects "ModuleRespawnPositionWest_F";
```
Comment (CreepPork) [inline src/ZGM-17_Achilles_Blufor.Altis/initPlayerLocal.sqf]: What if the mission is for OPFOR or Indeps?

**Review by CreepPork — CHANGES_REQUESTED** [review]

File: src/ZGM-17_Achilles_Blufor.Altis/initPlayerLocal.sqf (hunk 4 above)
Comment (Radiumio) [reply src/ZGM-17_Achilles_Blufor.Altis/initPlayerLocal.sqf]: I don't remember editing any other code other than the code I added for the rules etc itself. This was untouched. ¯\_(ツ)_/¯

File: src/ZGM-17_Achilles_Blufor.Altis/initPlayerLocal.sqf (hunk 1 above)
Comment (Radiumio) [reply src/ZGM-17_Achilles_Blufor.Altis/initPlayerLocal.sqf]: Would it suffice just having `execVM "scripts\rules.sqf";` alone then?

Comment (CreepPork) [reply src/ZGM-17_Achilles_Blufor.Altis/initPlayerLocal.sqf]: Yes, it should.

File: src/ZGM-17_Achilles_Blufor.Altis/initPlayerLocal.sqf [hunk 5]
```diff
@@ -44,6 +44,5 @@ if (_is_curator) then
 	execVM "scripts\R3F_LOG\init.sqf";
 };
 // initalise diary record for rulesinit
-if (hasInterface) then {
-	execVM "scripts\rules.sqf";
+execVM "scripts\rules.sqf";
 };
```
Comment (CreepPork) [inline src/ZGM-17_Achilles_Blufor.Altis/initPlayerLocal.sqf]: Syntax error.

**Review by CreepPork — CHANGES_REQUESTED** [review]

File: src/ZGM-17_Achilles_Blufor.Altis/initPlayerLocal.sqf (hunk 4 above)
Comment (CreepPork) [reply src/ZGM-17_Achilles_Blufor.Altis/initPlayerLocal.sqf]: Then you should revert it as it was.

Comment (Radiumio) [reply src/ZGM-17_Achilles_Blufor.Altis/initPlayerLocal.sqf]: That's the problem, that was the way I found it, and is the same on the unedited copy I have on my desktop. This is a BLUFOR mission anywho, so wouldn't we have to just change this line for other mission sides?

Comment (CreepPork) [reply src/ZGM-17_Achilles_Blufor.Altis/initPlayerLocal.sqf]: It uses this BLUFOR mission template and makes copies for all sides. Use the GitHub diff to help you to revert the changes.

**Review by CreepPork — APPROVED** [review]

**Review by Kexanone — APPROVED** [review]

**Pull request merged**